### DISCUSSION TITLE
Keep Playing Music when Window is Inactive?

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#351] [General]
Is there a way to set Exult to keep playing the music even when the window is not active in Windows?

I have a set of wireless headphones that turn off when there is no sound for 5 minutes, so I want something playing at low volume in the background. Then I can play boss music when my boss chats with me.

**Comment by wench** [comment]
Exult has a config setting for this. In exult,cfg set `config/audio/disablepause` to yes. I believe the setting may also be in the in-game menu
